src/Runner.Worker/FileCommandManager.cs

```diff
@@ -182,35 +185,75 @@ public void ProcessCommand(IExecutionContext context, string filePath, Container
                     return;
                 }
 
-                if (fileSize > AttachmentSizeLimit)
-                {
-                    context.Error(String.Format(Constants.Runner.UnsupportedSummarySize, AttachmentSizeLimit / 1024, fileSize / 1024));
-                    Trace.Info($"Step Summary file ({filePath}) is too large ({fileSize} bytes); skipping attachment upload");
+                Trace.Verbose($"Step Summary file exists: {filePath} and has a file size of {fileSize} bytes");
```

Comment:
you can keep the change in your fork repo and not create PR to the main repo if the changes are not ready for review. 😄

Reply:
Ah great, I will close this PR for now, then 😄